### PR TITLE
Only read and parse .js and .tsx files in the commands folder

### PR DESCRIPTION
**Pull request by zimme**
I accidentally put my `tsconfig.json` file in my `commands` folder and I saw that i got a `cli tsconfig.json` command which crashes the cli once I run it.

This PR will filter out any files in `commands` that is not a `.js` file or `.tsx` file.

This is based on #20. If you want to add this without the typescript support just remove the `.tsx` ext in this PR first.

### CODE REVIEW

**Comment by vadimdemedes** [comment]
Could you update this PR, so that we can see a clear diff without changes that are already in master?

**Comment by zimme** [comment]
Once #20 is merged, the diff will be clear. Some of the changes in here are using typescript stuff.

**Comment by zimme** [comment]
Aah ok, you're using rebase/squash and merge :D

I'll rebase on master and push again

File: lib/read-commands.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const readCommands = async (commandsPath, dirPath) => {
 	const paths = fs.readdirSync(dirPath);
 	const commands = [];
 
-	const promises = paths.map(async path => {
+	const promises = paths.filter(path => ['.js', '.tsx'].find(ext => path.endsWith(ext))).map(async path => {
```
Comment (vadimdemedes) [inline lib/read-commands.js]: I think this can be simplified as just `path.endsWith('.js') || path.endsWith('.tsx')`.

Comment (sw-double) [reply lib/read-commands.js]: `.some()` would be more appropriate here instead of `find()`

**Pull request merged**